lmcache/experimental/lookup_server/redis_server.py

```diff
@@ -12,13 +15,29 @@
 
 logger = init_logger(__name__)
 
+BATCH_SIZE = 10
```

Comment:
The tuning of these parameters should be benchmark driven, so it's best to make them configurable from env.

Reply:
While we're here, maybe now is the right time to split the configuration of sharing from indexing properly? (instead of the port hack). Not a PR blocker.